lib/OpenQA/WebAPI/Plugin/Helpers.pm

```diff
@@ -185,6 +185,24 @@ sub register {
             }
         });
 
+    $app->helper(
+        # Just like 'include', but includes the template with the given
+        # name from the correct directory for the 'branding' config setting
+        # falls back to 'plain' if brand doesn't include the template, so
+        # allowing partial brands
+        include_branding => sub {
+            my ($c, $name) = @_;
+            my $path = "branding/" . $c->app->config->{'global'}->{'branding'} . "/$name";
+            my $ret  = $c->render_to_string($path);
```

Comment:
and perlcritic doesn't seem to have such a policy - but you can implement own policies